src/Mail/SmtpMailer.php

```diff
@@ -122,6 +126,12 @@ protected function connect()
 		if (!$this->connection) {
 			throw new SmtpException($error, $errno);
 		}
+
+		// set context options
+		if ($this->context !== array()) {
+				stream_context_set_option($this->connection, $this->context);
+		}
```

Comment:
* the comment is useless,
* the condition should be just `if ($this->context) {`
* the indentation is wrong